README.md

```diff
@@ -1,24 +1,24 @@
-# Starter Redux assessment: Doggiegram
+# Starter redux assessment: Doggiegram
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## Available Scripts
+## Available scripts
 
-In the project directory, you can run:
+In the project directory, you can run the following commands:
 
 ### `npm install`
 
-Installs the project dependencies, including Redux packages such as "@reduxjs/toolkit" and "react-redux".
+Installs the project dependencies, including Redux packages such as @reduxjs/toolkit and react-redux.
 
 ### `npm run dev`
 
 Runs the React app in the development mode.\
 Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
 
-Runs an Express API at `http://localhost:3004` that exposes a single endpoint, GET `/api/suggestion`, which returns a dog suggestion at random.
+Runs an Express API at `http://localhost:3004` that exposes a single endpoint, `GET /api/suggestion`, which returns a dog suggestion at random.
 
 The page will reload when you make changes.\
-You may also see any lint errors in the console.
```

Comment:
"lint" is actually the correct term here. We could actually just remove this line.

Reply:
Fixed and ready for merging! Thanks @kchia.